packages/react-magma-dom/src/components/StyledButton/styles.ts

```diff
@@ -297,28 +367,45 @@ export function buildActiveColor(props) {
   props = buildPropsWithDefaultButtonStyles(props);
 
   if (props.color === 'marketing') {
+    if (props.variant !== 'solid' && props.isInverse) {
+      return props.theme.colors.tertiary500;
+    }
     return props.theme.colors.foundation02;
```

Comment:
Let's remove the usage of this old color

Reply:
Done